cnxeasybake/oven.py

```diff
@@ -386,7 +402,7 @@ def build_recipe(self, element, step, depth=0):
             if 'outside_deferred' in matching_rules:
                 for rule, declarations in \
                         matching_rules.get('outside_deferred'):
-                    logger.debug(u'Rule ({}): {}'.format(*rule))
+                    self.report_coverage(rule)
```

Comment:
hehe, I just complained about my own copy pasta! That repetition was there in the debug, now I see. What was one loop until I added pseudoelement support.